funnel/forms/profile.py

```diff
@@ -38,6 +38,11 @@ class ProfileForm(OrganizationForm):
             image_url_validator(),
         ],
     )
+    website = forms.URLField(
+        __("Website URL"),
+        description=(""),
```

Comment:
Did you mean `__("")`? Also, description kwarg can be skipped if there isn't one.